WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/builders/ScanStateListItemsBuilderTest.kt

```diff
@@ -187,14 +195,96 @@ class ScanStateListItemsBuilderTest : BaseUnitTest() {
             "<b>${threats.size}</b>",
             "<b>${site.name ?: resourceProvider.getString(R.string.scan_this_site)}</b>"
         )
+    }*/
+
+    @Test
+    fun `builds shield icon with green color for provisioning scan state model`() {
+        val scanStateModelInProvisioningState = scanStateModelWithNoThreats.copy(state = State.PROVISIONING)
+
+        val scanStateItems = buildScanStateItems(scanStateModelInProvisioningState)
+
+        assertThat(scanStateItems.filterIsInstance(IconState::class.java).first()).isEqualTo(
+            IconState(
+                icon = R.drawable.ic_shield_white,
+                colorResId = R.color.jetpack_green_5,
+                sizeResId = R.dimen.scan_icon_size,
+                marginResId = R.dimen.scan_icon_margin,
+                contentDescription = UiStringRes(R.string.scan_state_icon)
+            )
+        )
+    }
+
+    @Test
+    fun `builds preparing to scan header for provisioning scan state model`() {
+        val scanStateModelInProvisioningState = scanStateModelWithNoThreats.copy(state = State.PROVISIONING)
+
+        val scanStateItems = buildScanStateItems(scanStateModelInProvisioningState)
+
+        assertThat(scanStateItems.filterIsInstance(HeaderState::class.java).first()).isEqualTo(
+            HeaderState(UiStringRes(R.string.scan_preparing_to_scan_title))
+        )
+    }
+
+    @Test
+    fun `builds provisioning description for provisioning scan state model`() {
+        val scanStateModelInProvisioningState = scanStateModelWithNoThreats.copy(state = State.PROVISIONING)
+
+        val scanStateItems = buildScanStateItems(scanStateModelInProvisioningState)
+
+        assertThat(scanStateItems.filterIsInstance(DescriptionState::class.java).first()).isEqualTo(
+            DescriptionState(UiStringRes(R.string.scan_provisioning_description))
+        )
+    }
+
+    @Test
+    fun `builds empty list for unknown scan state model`() {
+        val scanStateModelInUnknownState = scanStateModelWithNoThreats.copy(state = State.UNKNOWN)
+
+        val scanStateItems = buildScanStateItems(scanStateModelInUnknownState)
+
+        assertThat(scanStateItems).isEmpty()
+    }
+
+    @Test
+    fun `builds empty list for unavailable scan state model`() {
+        val scanStateModelInUnAvailableState = scanStateModelWithNoThreats.copy(state = State.UNAVAILABLE)
+
+        val scanStateItems = buildScanStateItems(scanStateModelInUnAvailableState)
+
+        assertThat(scanStateItems).isEmpty()
+    }
+
+    @Test
+    fun `builds initial scanning description for scanning scan state model with no initial recent scan`() {
```

Comment:
Minor (🔍): Consider adding empty lines to split the test visually into the `Arrange/Act/Assert` sections.

PS: The same applies for the test further below.

Reply:
Nice catch, added in ecf37ba.

Reply:
🙏